tests/poppler/poppler/gen-unicode-tables.py

```diff
@@ -11,6 +11,11 @@
 HANGUL_S_BASE = 0xAC00
```

Comment:
This code is not written by us?  If its upstream code then better to at it to the exclude list.

Reply:
Done.